ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py

```diff
@@ -172,6 +177,23 @@ def __init__(self):
             self._sm_thread = Thread(target=self.sm_publisher, name="publish_sm")
             self._sm_thread.daemon = True
             self._sm_thread.start()
+        if self._rm_depth_AHAT_topic != DISABLE_TOPIC_STR:
+            # Create frame publisher
+            self.ros_frame_publisher = self.create_publisher(
```

Comment:
This shares the same name as the RGB image publisher, so this needs to be changed to be separate/unique.